gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/NewEntryPointEntity.java

```diff
@@ -54,7 +65,7 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (!(o instanceof NewEntryPointEntity)) return false;
         NewEntryPointEntity that = (NewEntryPointEntity) o;
-        return Objects.equals(value, that.value) && Objects.equals(tags, that.tags);
+        return Objects.equals(target, that.target) && Objects.equals(value, that.value) && Objects.deepEquals(tags, that.tags);
```

Comment:
Why are you using "deepEquals" here ?

Reply:
because I'm adding a new determinant and 2 identical Entity couldn't target the same target. in theory

Reply:
So we should also use `deepEquals` here: https://github.com/gravitee-io/gravitee-api-management/pull/11436/files#diff-06590cc8ae11f54756dda68bc70a870eead8729065d5bf165afdba43254d1e2cR82

Reply:
I don't really understand. is it a request for a refactor or have I forgotten something?  🤙  ?

Reply:
ho maybe i understand ^^ morning...  haha